newrelic/core/environment.py

```diff
@@ -170,6 +170,20 @@ def environment_settings():
         if hasattr(uvicorn, "__version__"):
             dispatcher.append(("Dispatcher Version", uvicorn.__version__))
 
+    if not dispatcher and "hypercorn" in sys.modules:
+        dispatcher.append(("Dispatcher", "hypercorn"))
+        hypercorn = sys.modules["hypercorn"]
+
+        if hasattr(hypercorn, "__version__"):
+            dispatcher.append(("Dispatcher Version", hypercorn.__version__))
+        else:
+            try:
+                from importlib.metadata import version
```

Comment:
I'm not sure if this is an issue or not cause I haven't tested it. I'm guessing probably not since the tests are passing but since there is a try pass here I just wanted to bring this up in case it might be an issue:
`importlib` is not available in python versions earlier than 3.8 so would there ever be a case here where `hypercorn.__version__` does not exist on python versions earlier than 3.8?